Documentation/api.md

```diff
@@ -226,7 +226,7 @@ NO+p24BL3PHZyKw0nsrm275C913OxEVgnNZX7TQltaweW23Cd1YBNjcfb3zv+Zo=
 
 ## Assets
 
-If you need to serve static assets (e.g. kernel, initrd), `matchbox` can serve arbitrary assets from the `-assets-path`.
+If you need to serve static assets, such as kernel, initrd, and so on, `matchbox` can serve arbitrary assets from the `-assets-path`.
```

Comment:
Haven't seen these usage before : "a kernel" , "a initrd". If you look at the description of initrd, for example, here http://www.tldp.org/LDP/Linux-Filesystem-Hierarchy/html/initrd.html  or in this manpage http://man7.org/linux/man-pages/man4/initrd.4.html